docs/release/trg-6/trg-6-1.md

```diff
@@ -0,0 +1,104 @@
+---
+title: TRG 6.01 - Released Helm Chart
+---
+
+| Author                | Status | Created     | Post-History    |
+|-----------------------|--------|-------------|-----------------|
+| Catena-X System Team  | Active | 30-Nov-2022 | Initial release |
+
+## Description
+
+A released Helm chart for each Tractus-X sub-product is expected to be available in corresponding GitHub repository.
+
+## Why
+
+Tractus-X components can only be used and bundled with other components of Tractus-X if there is a version of the
+component with reliable content. This can only be achieved by creating a release of your Helm chart. We recommend to use
+GitHub Chart Releaser Action.
+
+### Process description
+
+If you follow our recommendation to use GitHub Chart Releaser Action to provide a released Helm Chart for your Tractus-X
+sub-product, the process will provide the following for you:
+
+1. A GiHub Release is created with
+    - Release name _Chartname-Semver_ (e.g. _productABC-0.1.1_)
+    - A Git tag _Chartname-Semver_
+2. A local Helm repository is created (file `index.yaml` in branch _gh-pages_)
+
+:::caution
+
+Only if your follow this TRG to publish your Helm Chart, your released Helm chart will be automatically added to
+the [central Tractus-X Helm repository](https://eclipse-tractusx.github.io/charts/).
+
+:::
+
+### Prerequisites
+
+To get the GitHub Chart Releaser Action working, the following prerequisites must be met:
+
+- GitHub Pages must be enabled for the GitHub repository
```

Comment:
This is only partially true. This is only needed, if you want to transform your git repository to a Helm repository. 
The release itself and our workflow of collecting this to central helm repo does work without GitHub pages.
What is necessary is, that a branch named `gh-pages` does exist

Reply:
Removed requirement for GH Pages to be enabled.